modules/tinymce/src/plugins/importcss/main/ts/core/ImportCss.ts

```diff
@@ -33,6 +33,8 @@ interface Group extends UserDefinedGroup {
   readonly filter: Filter | undefined;
 }
 
+const internalEditorStyle = /^\.(?:ephox|tiny|mce)(?:-+\w+)+$/;
```

Comment:
I sort of hinted at this on the Jira, but should have been more explicit sorry but `tiny` is a common term so I don't think we can do this. For example, this will match `tiny-image` or `tiny-text`, so unfortunately I think we're going to have to use `tiny-pageembed` explicitly here even though it's not very generic or future proof (really we should never use `tiny` for internal class names).

- [ ] Reconsider matching on `tiny-`.

Reply:
hmm. I don't think it'll be _that_ much of a problem, it would need to be a customer using styles like that _and also_ the import css plugin (which I don't believe is very popular). But if we're going to avoid using `tiny-*` class names in future this makes sense.

Reply:
The problem is we just really have no actual data on what is being used, so this change could completely break someones integration. We definitely should avoid using `tiny-` prefixed class names for internal plugins in future though and stick to either `ephox` or `mce` prefixes.